### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by manasa2107**
This is restaurant application having menu items and table items where some functionalities like drag and drop, searching for items and generating bill are performed.

### CODE REVIEW

File: billTable.js [hunk 1]
```diff
@@ -0,0 +1,150 @@
+let currTable, itemCount, targetId, orderData;
+let tableHeader=document.querySelector(".table-name");
+let popUp=document.querySelector(".table-list");
+let totalTables=document.querySelectorAll(".table-item");
+let billAmount=document.getElementById("total");
+let tableBody = document.querySelector(".bill");
```
Comment (SravaniKusuma) [inline billTable.js]: can you use const over let if the value is not changing?
can you also replace everywhere in the code ?

File: billTable.js [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+      orderDetails = tableList.get(eachTable.id).itemList;
+      for (let [item, count] of orderDetails) {
+        tableBody.append(addRows(item, count));
+      }
+    });
+  }
+
+  //to add blur effect in background
+
+  function toggle(){
```
Comment (SravaniKusuma) [inline billTable.js]: try to follow ES6 syntax and use arrow functions wherever required

File: billTable.js [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+    var blur=document.getElementById("blur");
+    blur.classList.add("active");
+    var pop=document.getElementById("pop")
```
Comment (SravaniKusuma) [inline billTable.js]: use const or let try to avoid using var

**Comment by SravaniKusuma** [comment]
use ES6 syntax wherever required and some best practices like using const, and ...
I have seen there is no use of session storage or so to generate bill, how is that even handled @manasa2107

File: billTable.js (hunk 1 above)
Comment (manasa2107) [reply billTable.js]: I made the respective changes .

**Review by SravaniKusuma — APPROVED** [review]

**Pull request merged**